bin/client/main.rs

```diff
@@ -91,6 +92,37 @@ async fn main() -> std::result::Result<(), ClientError> {
                 .await?;
             println!("Invoice {} created", resp.invoice.bindle.id);
         }
+        SubCommand::SignInvoice(sign_opts) => {
+            // Role
+            let role = if let Some(r) = sign_opts.role {
+                role_from_name(r)?
+            } else {
+                SignatureRole::Creator
+            };
+            // Keyfile
+            let keyfile = sign_opts.secret_file.unwrap_or_else(|| {
+                default_config_dir()
+                    .unwrap_or_else(|| PathBuf::from("."))
```

Comment:
Nit: I think you can just do `PathBuf::from("secret_keys.toml")` here

Reply:
I don't think so... that would give me `secret_keys.toml/secret_keys.toml`. The idea is to get the default config dir or else use `.`, and _then_ append `secret_keys.toml`. So if config dir is `/foo/bar`, the result should be `/foo/bar/secret_keys.toml`.

Or maybe I'm misunderstanding something about what you are suggesting?

Reply:
Oh right, you are joining on the result of the operation. Ignore me